LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaPayloadDecoder.cs

```diff
@@ -56,7 +56,19 @@ public async ValueTask<DecodePayloadResult> DecodeMessageAsync(string devEUI, by
             var base64Payload = ((payload?.Length ?? 0) == 0) ? string.Empty : Convert.ToBase64String(payload);
 
             // Call local decoder (no "http://" in SensorDecoder)
-            if (!sensorDecoder.Contains("http://", StringComparison.Ordinal))
+            if (Uri.TryCreate(sensorDecoder, UriKind.Absolute, out var url) && (url.Scheme is "http" or "https"))
```

Comment:
Was it intended to allow https now?

Reply:
I found it harmless to add but for sake of 100% backward compatibility, I could remove it. What's your take?

Reply:
Can't see a problem with being more permissive and allow https unless we have trouble handling it. but also not very familiar with different decoders so can't give a very educated opinion here :)

Reply:
Reverted to HTTP-only with 75af256a04566d254f0548123e2e7aaf939d99c9.